src/emucore/OSystem.cxx

```diff
@@ -128,10 +128,17 @@ bool OSystem::create()
   // Create the event handler for the system
   //ALE   myEventHandler = new EventHandler(this);
   //ALE  myEventHandler->initialize();
-  
   // Create the streamer used for accessing eventstreams/recordings
+
+  // Delete the previous event object (if any).
+  delete myEvent;
```

Comment:
No, yes, and not "now" but "has always accepted".

I wasn't going to put any work into this entirely dead branch of code, as we discussed previously, since mainline Stella has moved on far beyond this. Do you care to put additional maintenance work into this branch?